test/studies/lcals/LCALSDataTypes.chpl

```diff
@@ -322,8 +322,54 @@ module LCALSDataTypes {
     var RealArray_2D_Nx25: [0..#s_num_2D_Nx25_Real_arrays][0..#aligned_chunksize, 0..#25] real;
     var RealArray_2D_7xN: [0..#s_num_2D_7xN_Real_arrays][0..#7, 0..#aligned_chunksize] real;
     var RealArray_2D_64x64: [0..#s_num_2D_64x64_Real_arrays][0..#64, 0..#64] real;
-    var RealArray_3D_2xNx4: [0..#s_num_3D_2xNx4_Real_arrays][0..#2, 0..#aligned_chunksize, 0..#4] real;
+
+    //var RealArray_3D_2xNx4: [0..#s_num_3D_2xNx4_Real_arrays][0..#2, 0..#aligned_chunksize, 0..#4] real;
```

Comment:
I left this one in on purpose.  The commented out line is how it seems like that array _should_ be declared, but instead it uses the LCALS_Overlapping_Array_3D.  I can remove it if you'd prefer.